content/SCALE/SCALE22.12.md

```diff
@@ -52,6 +52,21 @@ Manual update files are also available at this location.
 
 To upgrade an existing SCALE install, log in to your SCALE web interface and go to **System Settings > Update**.
 
+## 22.12.3.3
+**July 25, 2023**
+
+iXsystems is pleased to release TrueNAS SCALE 22.12.3.3!
+
+This is a small hotpatch to address kernel panics that appear in specific circumstances when files in an encrypted pool have differing indirect block size (IBS) values and a replication task configured for that pool starts.
+
+### 22.12.3.2 Changelog
+
+* [NAS-122583](https://ixsystems.atlassian.net/browse/NAS-122583) Crash on ZFS replication receive with different indirect block size
```

Comment:
My 0.02, but this ticket is probably all that needs to be linked since the other 2 are about the same issue. 1 of the tickets (NAS-122883) is actually marked as a duplicate of NAS-122583 so not sure if we want to include 3 tickets when they all just reference 1 fix.